housekeeper/cli/delete.py

```diff
@@ -134,6 +134,28 @@ def files_cmd(context, yes, tag, bundle_name, before, notondisk, list_files, lis
             delete_file(file=file, store=store)
 
 
+@delete.command("tag")
+@click.option("-n", "--name", help="Name of the tag to delete")
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.pass_context
+def tag_cmd(context, yes, name: str):
+    """Delete a tag from the database.
+    Entries in the file_tag_link table associated with the tag will be removed."""
```

Comment:
Is this something the caller needs to know or is it an implementation detail that can be omitted?

Reply:
I think it is better to have it just in case. It does not add a lot of volume to the function or the help @seallard